PROTO_tests/tests/numeric_test.py

```diff
@@ -10,6 +10,12 @@
 INT_FLOAT = [ak.int64, ak.float64]
 CAST_TYPES = [ak.dtype(t) for t in ak.DTypes]
 
+NP_TRIG_ARRAYS = {
+    ak.int64: np.arange(0, 10, dtype=np.int64),
+    ak.float64: np.linspace(0, 10),
+    ak.uint64: np.arange(0, 10, dtype=np.uint64),
+}
```

Comment:
this doesn't pick as good of representative examples as I was looking for. We don't get any negative ints or floats, uint's never go into the `> 2**63` range, and we don't test any float edge cases